.pipelines/publish-to-pypi.yml

```diff
@@ -0,0 +1,33 @@
+variables:
```

Comment:
taken from https://github.com/microsoft/presidio/blob/main/.pipelines/templates/publish-to-pypi.yml